2010-wolesi-jirga/candidates.csv

```diff
@@ -1,2505 +1,2505 @@
-7466501,F,1,پوهنیار  شیلا  بابړی ,7
-7476851,M,1,مامور عبدالجبار شلگری,47
-7477101,M,1,حاجی ملک نظیر ,38
-7477213,M,1,داود شاه خوشحال,134
-7533130,M,1,حاجی ګل محمد ,28
-7533981,M,1,حاجی سید ولی څمکنی ,34
-7541732,M,1,عبدالشکور,27
-7542812,M,1,حیات وزیر ,25
-7543344,F,31,مریم دقیق,9
-7554179,M,1,محمد ظاهر غنی زاده مشهور به قند آغا,70
-7557536,M,54,حاجی عبدالوهاب,50
-7564567,M,1,الحاج محب الله محقق,116
-7571246,F,1,فوزیه هوتک ,24
-7595590,M,1,,
-7609105,M,1, شرف الدین شینواری,126
-7621064,M,1,داکتر عصمت الله  شینواری,45
-7622400,M,1,حاجی پیربخش  گردیوال ,97
-7637873,M,1,ډاکتر حبیب الله  زاخیلوال ,101
-7647079,M,1,شفاګل   انګار,30
-7647395,M,1,لیاقت الله بابکرخیل,9
-7648001,M,1,کریم خیل ,45
-7692380,M,1,حاجی نیاز محمد سرحدی,5
-7709513,M,1,الحاج محمداسحق رهگذر ,16
-7709527,F,1,داکتر گلالی نور صافی,27
-7722209,M,1,حاجی اصیل خان هوتک,50
-7726013,F,1,شاه گل رضایی,17
-7726020,F,1,هوسی اندړ,31
-7726038,M,2,سید محمود حسام الدین گیلانی,24
-7726040,M,1,حاجی نیاز محمد امیری,40
-7727504,M,1,سید حسام الدین ابراهیمی,5
-7727505,F,1,استاد چمن حمیدی,18
-7727510,F,1,عزیزه جلیس,4
-7727513,M,1,محمد صادق مخدوم,19
-7727514,M,1,محمد حسین فهیمی,26
-7731121,M,1,حاجی عصمت الله   فیض  ,9
-7732202,M,1,څارنوال عزیزالرحمن,42
-7732203,F,1,استاد کریمه صدیقی,44
-7732205,M,1,انجینر شمس الرحمن شمس,17
-7732206,M,1,سید ذبیخ الله عباس,59
-7732207,F,1,داکتر ذکیه قدرت جلیلی,50
-7732208,M,1,داکتر همایون   لغمانی,11
-7732209,M,1,محمد اغا   اوریا خیل,25
-7732210,M,1,خسرو یوسفزی,7
-7732211,M,1,داکترمحمد هارون سیلاب  عبدالرحیمزی,26
-7732212,M,102,مولوی عبدالحلیم  حلیمی,39
-7732213,M,1,مولوی حیات الله حیات فرقانی پشه ئی,57
-7732214,F,1,شیرین تاج      سهاک,22
-7732215,F,1, اوگی امیل,34
-7732216,M,1,عبدالغنی  اتمر,23
-7732217,M,1,غفران الله,53
-7732218,M,1,محمد همایون حقمل      کمالپوری,16
-7732219,M,1,محمد اسماعیل    نشاط,37
-7732220,M,1,ذبیح الله بهار غازی,52
-7732221,M,1,عبدالرشید عمر خیل,40
-7732222,M,1,صفی الدین مدثر لغمانی,10
-7732223,M,1,الحاج حفیظ احمد گلزاد,43
-7732224,M,1,مولوی سعیدالرحمن,49
-7732225,M,1,محمد داود نیازی ,19
-7732226,M,1,انجینرمحمد افضل افضل زاده,27
-7736015,M,1,میرویس یاسینی,39
-7739182,M,1,اسدالله مصطفی ,5
-7762167,F,1,فوزیه ناصر یار گلدره ئی,24
-7762310,M,1,سفیر احمد دانش,30
-7765673,M,1,بریالی گلیار,10
-7767635,M,1,حاجی محمد حسن  مموزی,41
-7770507,M,1,پوهیالی الحاج سید عبدالحبیب هاشمی سمونوال متقاعد,305
-7771232,M,1,احمد الله  فیاض,91
-7783285,F,1,بی بی حاجی وکیله شفیقه نوری,57
-7784763,M,1,,
-7787021,M,1,شاه ولی  منګل,25
-7787038,M,1,دوست محمد خان ځدران ,55
-7787057,M,1,ډاکټر  باختر  امین زی,49
-7810007,F,1,رابعه لامع,503
-7820210,M,1,حاجی نور محمد عبیدی,362
-7820258,M,1,ذبیح الله جوانمرد ,348
-7820324,M,1,حاجی دولت محمد شیرزی,300
-7821642,M,1,الحاج غلام ربانی ناشر ,79
-7827841,M,1,محمد اکبر صافی,159
-7829886,F,1,کریمه بیهقی,43
-7832817,F,1,جمیله مجاهد,285
-7832834,F,1,شیلا صبا,473
-7835492,M,1,محمد جواد باذل,652
-7889725,M,1,,
-7900041,M,1,الحاج فرید احمد شیرزاد,26
-7902051,F,1,فهیمه نیرو,84
-7907498,M,1,امیر محمد,20
-7931025,M,1, حاجی وحید صدیقی ,46
-7931494,M,1
```

Comment:
The header for `name_dari` and `ballotNumber` seem to be switched around.
Also, can you make it `nameDari` for consistency?

Reply:
Scrap this. The git diff showed a different column order than the actual file has. This can be merged.